src/hashing/aahash_iterator.rs

```diff
@@ -107,6 +111,16 @@ impl AaHashIterator {
         hash_vec
     }
 
+    pub fn from_3di_file(file: &str, concat_fasta: bool) -> Vec<Self> {
+        Self::new(file, AaLevel::Level1, concat_fasta)
```

Comment:
`concat_fasta` I think would always be false for a PDB file?

Reply:
Yes, modified this, but waiting for confirmation on the other comment